hazelcast/src/main/java/com/hazelcast/query/impl/IdentityConverter.java

```diff
@@ -21,10 +21,10 @@
 /**
  * Converts to the same value
  **/
-public class IdentityConverter implements TypeConverter {
+public class IdentityConverter extends TypeConverter {
```

Comment:
As far as I see, with this PR, `IdentityConverter` is only access over `TypeConverters.IDENTITY_CONVERTER`, so should we keep this still public?
Can it cause to backward-compatibility issue if we remove `public` class modifier?